### ISSUE TITLE
"Delete All" button in IDEA should also clear breakpoint icons.

### ISSUE THREAD

**Issue by joaoandremartins** [GoogleCloudPlatform/gcloud-intellij#437]
The button converts active snapshots into inactive, but it leaves the previously hit breakpoint icons there.

What should the behaviour be? I would want the icons to all go away and start with a fresh environment so that I can add new breakpoints with a single click, instead of clicking twice, once to remove the old icon and twice to add the new one.

**Comment by patflynn** [comment]
Yeah, I'm seeing the snapshot location still in the margin even though it's
deleted from the snapshot list in the debug view.

It sounds like the right behavior is that they should be completely removed
everywhere.
